tools/ms/openssl_setup.cmd

```diff
@@ -1,8 +1,8 @@
 @echo off
 setlocal EnableDelayedExpansion
-
 if [%1]==[] goto usage
-
+set SDL_OPENSSL_DIR="C:\OpenSSL\openssl_win_x64"
```

Comment:
@antongovorov,
Why we are hardcode the path?
Maybe better to use System path or CMAKE parameter?